src/styles/index.scss

```diff
@@ -1,3 +1,88 @@
+$stopwatcDmtr: 80vmin;
+$centerDmtr: 5vmin;
+$secondsW: 1.5vmin;
+$secondsH: 38vmin;
+$minutesW: 3vmin;
+$minutesH: 20vmin;
+
+@mixin animation($duration, $timing) {
+  animation-name: rotate;
+  animation-duration: $duration;
+  animation-direction: normal;
+  animation-iteration-count: infinite;
+  animation-timing-function: $timing;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: $stopwatcDmtr;
+  height: $stopwatcDmtr;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    top: $stopwatcDmtr / 2 - $minutesH;
+    left: $stopwatcDmtr / 2 - $minutesW / 2;
+
+    transform-origin: $minutesW / 2 $minutesH;
+
+    @include animation(3600s, steps(60, end));
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
```

Comment:
The width value here is hardcoded as `1.5vmin`, but you have a variable `$secondsW` defined. Consider using `$secondsW` for consistency and maintainability.